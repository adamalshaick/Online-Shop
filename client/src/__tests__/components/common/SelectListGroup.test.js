import React from "react";
import SelectListGroup from "../../../components/common/SelectListGroup";
import renderer from "react-test-renderer";

const Props = {
  options: []
};

const wrapper = renderer.create(<SelectListGroup {...Props} />).toJSON();

describe("render component", () => {
  it("renders component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
