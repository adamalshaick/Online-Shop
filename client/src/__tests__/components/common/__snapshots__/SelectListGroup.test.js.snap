// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render component renders component 1`] = `
<select
  className="form-control form-control-lg"
/>
`;
